src/popup/components/LatestTransactionsCard.vue

```diff
@@ -34,8 +34,8 @@ import {
   watch,
   ref,
 } from 'vue';
-import { useStore } from 'vuex';
 import { useRoute } from 'vue-router';
+import { useStore } from 'vuex';
```

Comment:
This change gives no value but increase the possibility of future conflicts. I suggest to not apply this change.